packages/shared-ux/button_toolbar/src/popover/popover.tsx

```diff
@@ -10,9 +10,9 @@ import React, { useState } from 'react';
 import { EuiPopover } from '@elastic/eui';
 import { Props as EuiPopoverProps } from '@elastic/eui/src/components/popover/popover';
 
-import { PrimaryButton, Props as ButtonProps } from '../buttons/primary/primary';
+import { ToolbarButtonProps, ToolbarButton } from '../buttons';
 
-type AllowedButtonProps = Omit<ButtonProps, 'onClick' | 'fill'>;
+type AllowedButtonProps = Omit<ToolbarButtonProps, 'iconSide' | 'onClick' | 'fill'>;
```

Comment:
The `iconSide` prop will be determined depending on whether or not an `iconType` is provided - if it is, then the provided icon will be rendered on the left side of the button; if it isn't, then a `arrowDown` icon will be rendered on the right side of the button: